buffalo/cmd/destroy/resource.go

```diff
@@ -9,6 +9,7 @@ import (
 	"path/filepath"
 	"strings"
 
+	"github.com/Sirupsen/logrus"
```

Comment:
The repo is now lower case `sirupsen`. Can you please undo these import changes?